src/runtime_src/core/tools/xbutil2/OO_MemRead.cpp

```diff
@@ -105,38 +111,63 @@ OO_MemRead::execute(const SubCmdOptions& _options) const
     throw xrt_core::error(std::errc::operation_canceled);
   }
 
+  if (m_count <= 0) {
+    std::cerr << "ERROR: Please specify a number of blocks greater than zero\n";
```

Comment:
Hmm...in other examples in the code, an error message was added to the throw operation.  Can the same be done here?

Reply:
Same reason as this comment: https://github.com/Xilinx/XRT/pull/6491#discussion_r923815187.